front/src/routes/map/Map.jsx

```diff
@@ -84,6 +87,10 @@ class MapComponent extends Component {
   componentDidUpdate() {
     this.displayHouses();
     this.displayUsers();
+    if (markerArray.length >= 1) {
+      var group = leaflet.featureGroup(markerArray);
```

Comment:
`var` should not be used anymore, here `const` need to be used. 

I just realized that eslint was not checking .jsx file..  my bad!